src/commands/data/getFilesByCount/getFilesByCount.js

```diff
@@ -0,0 +1,62 @@
+import { validateApiKeys } from "../../../util/validators";
+import pinList from "../pinList/pinList";
+
+export default function getFilesByCount(
+    pinataApiKey,
+    pinataSecretApiKey,
+    maxCount,
+    filters
+) {
+    validateApiKeys(pinataApiKey, pinataSecretApiKey);
+    if (!maxCount) {
+        throw Error("Max count needs to be defined ");
+    }
+
+    const asyncIterable = {
+        [Symbol.asyncIterator]: () => {
+            let i = 0;
+            const pageLimit = 10;
+            let pageOffset = 0;
+            let cache = [];
+            let count = 0;
+            
+            return {
+                next() {
+                    return new Promise((resolve, reject) => {
+                        if (i === 0 || (i % pageLimit === 0 && i < count)) {
+                            resolve(
+                                pinList(pinataApiKey, pinataSecretApiKey, {
```

Comment:
@rommel-pinata we need to change the way the `pinList` call works before we can do this. The pinList method needs to include a query string of `includesCount=false`

Reply:
@polluterofminds  I added the includeCount param to the pinList!